examples/flex-flows/basic/flow.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import TypedDict
+
+from jinja2 import Template
+from llm import my_llm_tool
+
+from promptflow.tracing import trace
+
+BASE_DIR = Path(__file__).absolute().parent
+
+
+class Result(TypedDict):
+    output: str
+
+
+@trace
+def load_prompt(jinja2_template: str, text: str) -> str:
+    """Load prompt function."""
+    with open(BASE_DIR / jinja2_template, "r", encoding="utf-8") as f:
+        prompt = Template(
+            f.read(), trim_blocks=True, keep_trailing_newline=True
+        ).render(text=text)
+        return prompt
+
+
+@trace
+def flow_entry(text: str = "Hello World!") -> Result:
+    """Flow entry function."""
+    prompt = load_prompt("hello.jinja2", text)
+    output = my_llm_tool(
+        prompt=prompt, deployment_name="text-davinci-003", max_tokens=120
```

Comment:
can add an optional parameter of deployment name? Otherwise cannot run the sample notebook smoothly without having this deployment in the endpoint

Reply:
added. and change to chat completion api